README.md

```diff
@@ -314,6 +314,24 @@ extra_aliased_targets = [
 Cargo-raze prefixes binary targets with `cargo_bin_`, as although Cargo permits
 binaries and libraries to share the same target name, Bazel disallows this.
 
+#### Crates that only provide binaries
```

Comment:
Nit, I very very slightly prefer that this live with the feature PR, but it's probably more work to move this than I'd actually feel is justified.

Reply:
This change is only here because it includes the changes from #227 It'll be removed from this PR once that goes through